package.json

```diff
@@ -67,6 +67,7 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "broadcast-channel": "^3.4.1",
+    "jscodeshift": "^0.13.1",
```

Comment:
I don't think we should add this as a dependency? If the plan is to run the codemods via `npx jscodeshift`, it also shouldn't be necessary I believe?

Reply:
That was a mistake, I wanted to add it to the `devDependencies`. 😅 

980f30f9957a2c3681489d1a64e068f9861b8cde

Reply:
the linked commit only shows changes to `package.json` - is the `yarn.lock` file up to date? I can't see it, but it shouldn't have `jscodeshift` as dependency, but only as dev-dependency, right?